tests/S3/StreamWrapperTest.php

```diff
@@ -409,6 +409,16 @@ public function testCanDeleteNestedFolderWithRmDir()
 
     /**
      * @expectedException \PHPUnit_Framework_Error_Warning
+     * @expectedExceptionMessage rename(): Cannot rename a file across wrapper types
+     */
+    public function testRenameEnsuresProtocolsMatch()
```

Comment:
On the fence about this unit test since it's technically only unit testing native PHP functionality that's outside the scope of the StreamWrapper. Thoughts?